docs/interfaces/datastore-service.txt

```diff
@@ -119,70 +120,69 @@ Event ModifiedFieldsEvent on topic ModifiedFields {
  */
 getIds(collection: Collection, n: number): Id[]
 
+
 ## Reader
 # Note: Different host and port than the writer!
 
 /** Common notes:
  * - parameter `position`: Optional, if given reads the data to this position.
- * - parameter `mapped_fields`: List of fields, that should onl be present in the response.
+ * - parameter `mapped_fields`: List of fields that should only be present in the response.
+ * - parameter `get_deleted_models`: Optional, defines which models to return
+ *    - DeletedModelsBehaviour.NO_DELETED:   (Default) only non-deleted models are returned.
+ *                                           get throws a ModelDoesNotExist error if the given
+ *                                           model is deleted.
+ *    - DeletedModelsBehaviour.ONLY_DELETED: only deleted models are returned. get throws
+ *                                           a ModelNotDeleted if the given model is not deleted.
+ *    - DeletedModelsBehaviour.ALL_MODELS:   all models are returned
  * - All operations adds the fields `meta_position` and `meta_deleted` to the models.
  * - The InvalidFormat exception can always be thrown, if the requested formats are
  *   wrong, including something like empty collections, ...
  */
 
-/**
- * Returns a model. Deleted models are not returned (and handled as a ModelDoesNotExit)
- *
- * @throws ModelDoesNotExist
- * @throws InvalidFormat
- */
-get(model: Fqid, position?: Position, mapped_fields?: fields[]): Partial<Model>;
+Enum DeletedModelsBehaviour {
+    NO_DELETED,
```

Comment:
Please add values here, so a user know, hwat to send in a request (numbes or strings, which one is not important).